Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Lingmo OS/lingmo-settings](https://hosted.weblate.org/projects/lingmoos/lingmo-settings/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/lingmoos/lingmo-settings/horizontal-auto.svg)

Atalanttore committed:
Translated using Weblate (German)

Currently translated at 42.0% (117 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/de/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/LingmoOS/lingmo-settings?pullRequest=48) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/LingmoOS/lingmo-settings?pullRequest=48) before we can accept your contribution.<br/>**1** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: elysia-best<br/>:x: TamilNeram<br/>:x: Atalanttore<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/LingmoOS/lingmo-settings?pullRequest=48) it.</sub>

elysia-best committed:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 41.3% (115 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/zh_Hant/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (278 of 278 strings)

Translation: Lingmo OS/lingmo-settings
Translate-URL: https://hosted.weblate.org/projects/lingmoos/lingmo-settings/ta/
